src/modules/75-cf/components/FlagArchiving/__tests__/useRestoreFlagDialog.test.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC } from 'react'
+import { render, RenderResult, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { cloneDeep } from 'lodash-es'
+import { TestWrapper } from '@common/utils/testUtils'
+import mockFeature from '@cf/components/EditFlagTabs/__tests__/mockFeature'
+import type { Feature } from 'services/cf'
+import * as cfServices from 'services/cf'
+import useRestoreFlagDialog, { RestoreFlagDialogProps } from '../useRestoreFlagDialog'
+
+const openRestoreDialogBtn = 'Open Restore dialog'
+
+const WrapperComponent: FC<RestoreFlagDialogProps> = ({ flagData, queryParams, onRefetch }) => {
+  const openRestoreDialog = useRestoreFlagDialog({ flagData, queryParams, onRefetch })
+
+  return <button onClick={() => openRestoreDialog()}>{openRestoreDialogBtn}</button>
+}
+
+const archivedMockFeature = cloneDeep(mockFeature)
+archivedMockFeature.archived = true
+
+const renderComponent = (props: Partial<RestoreFlagDialogProps> = {}): RenderResult => {
+  const result = render(
+    <TestWrapper>
+      <WrapperComponent
+        flagData={archivedMockFeature as Feature}
+        onRefetch={jest.fn()}
+        queryParams={{
+          accountIdentifier: 'mockAccountIdentifier',
+          orgIdentifier: 'mockOrgIdentifier',
+          projectIdentifier: 'mockProjectIdentifier'
+        }}
+        {...props}
+      />
+    </TestWrapper>
+  )
+
+  return result
+}
```

Comment:
No real need to assign to a variable and then return that variable
```suggestion
const renderComponent = (props: Partial<RestoreFlagDialogProps> = {}): RenderResult =>
  render(
    <TestWrapper>
      <WrapperComponent
        flagData={archivedMockFeature as Feature}
        onRefetch={jest.fn()}
        queryParams={{
          accountIdentifier: 'mockAccountIdentifier',
          orgIdentifier: 'mockOrgIdentifier',
          projectIdentifier: 'mockProjectIdentifier'
        }}
        {...props}
      />
    </TestWrapper>
  )
```

Reply:
this doesn't seem to be a blocker but I can address it in the next PR